Sources/LiveKit/LiveKit.swift

```diff
@@ -35,6 +35,7 @@ public class LiveKit {
 
     public static let version = "1.0.0"
 
+    @available(*, deprecated, message: "Use Room.connect() instead, latest protocol version is not supported with this method.")
```

Comment:
:nit: `Use Room.connect() instead, protocol v8 and higher do not support this method`?

Reply:
got it !